### DISCUSSION TITLE
Survey catch

### DISCUSSION THREAD

**Discussion by brianlangseth-NOAA** [pfmc-assessments/canary_2023#20] [Ideas]
@okenk My first discussion: I think its cleaner than doing this all in the issue.

Relevant to #12 lets calculate the survey catches. It shouldn't be hard, nor timely. Then we have them. 

Whether to include them in the model is the next decision. Im inclined to not, but I think we should see how they included them in the .dat file in the last assessment as a second step. Running a sensitivity with them included to show no effect should be sufficient.

**Comment by okenk** [comment]
I agree we should calculate them (we have to do this for bridging anyway?). I am also inclined to not include them in the model, mostly because of the complications Ian brought up with forecasts and allocations.

I am actually not sure how to calculate them though. The data frame provided by `nwfscSurvey` does not have a column for state. What are the accepted latitudinal divisions? (I hope they are latitudes and not something more complicated?)

**Comment by brianlangseth-NOAA** [comment]
Confirmed about catches. 

We may need to add a column for state ourselves. Wallace has [some code](https://github.com/John-R-Wallace-NOAA/PacFIN_Logbook_Download_and_Cleanup/tree/master/Compare%20Raw%20and%20Processed%20Logbook%20to%20FIshticket%20Data) that I glanced at from the non-trawl logbook email from chantel that defines Oregon as < 46 + 16/60 and CA as < 42

Im surprised that there isn't some agency code in the survey package. I will look and see what we've done in the past

**Comment by brianlangseth-NOAA** [comment]
For our data moderates we used rename_survey_data.R in the data_moderate2021 repo that defined the split between Oregon and Washington at 46, which of course is different...lol. 

Looking it up is harder than I thought it would be and appears to land on a different number. [Based on law](https://app.leg.wa.gov/rcw/default.aspx?cite=43.58&full=true) it appears to be 46 degrees and 15 minutes so 46.25 decimal degress.

**Comment by brianlangseth-NOAA** [comment]
Found it in nwfscSurvey, [here](https://github.com/pfmc-assessments/nwfscSurvey/blob/e291e13beac13e28332e0706708bd53da644158f/R/GetStrata.fn.R#L50-L52) which used 46 as the default

**Comment by okenk** [comment]
Sounds like 46 and 46.25 are both defensible and have both been used by PEP (46 + 16/60 ~ 46.25). I will use 46.

**Reply by brianlangseth-NOAA** [reply]
I suggest 46.25 for catch because it is the actual value, though acknowledge it will have little effect other than possibly inducing a difference with last assessments values. 

I think using whatever is set up within the getStrata script is appropriate though for doing expansions. I added an issue in nwfscSurvey. Perhaps there is good reason for using 46 there.
